Formula/glib.rb

```diff
@@ -71,6 +72,10 @@ def install
               "Requires.private: #{libffi}/lib/pkgconfig/libffi.pc"
     end
 
+    inreplace Dir["#{bin}/{gdbus-codegen,gtester-report,glib-genmarshal,glib-mkenums}"],
```

Comment:
This seems like a very crude way to fix things… do we have another way?

Reply:
Found not so crude one: patching `meson.build`:

https://github.com/GNOME/glib/blob/ca9f51b82f57e05f6b3c82b98c235623afa47573/meson.build#L1996-L1997